packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -209,6 +209,96 @@ class SkiaGoldClient {
     return true;
   }
 
+  /// Executes the `imgtest init` command in the goldctl tool for tryjobs.
+  ///
+  /// The `imgtest` command collects and uploads test results to the Skia Gold
+  /// backend, the `init` argument initializes the current tryjob.
+  Future<void> tryjobInit() async {
+    final File keys = workDirectory.childFile('keys.json');
+    final File failures = workDirectory.childFile('failures.json');
```

Comment:
If I'm reading this right, it's coming from https://github.com/flutter/flutter/pull/44474/files#diff-259a52e89a62c0e0633f88f4307c22b7L142, which is not tracked by source control - is that right?

Reply:
> Is workDirectory in source control? 

I am not sure what you mean by this.

I thought it was automatically blown away on ci since we start with a clean instance for each build. I can split it out further for ci/non-ci code paths, but I thought this would keep it from becoming a more complicated code path for those cases.

Reply:
Yeah this should be fine.  I was worried we were modifying something that the Flutter source control cared about with this.